docs/breaking-changes/v4.md

```diff
@@ -0,0 +1,4 @@
+# Breaking Changes in v4
+
+The `golangcilint` hook will no longer be supported in favor of using the `golangci-lint` hook instead. The latter
+is put in place via the `packer-github-actions-general` repo.
```

Comment:
This is a public repo, so mentioning this repo here does not add much value

Reply:
Interesting, I did not notice this repo was Public. Can it be private? Should it instead live in `open-turo`?